rqd/rqd/rqmachine.py

```diff
@@ -188,8 +193,38 @@ def isUserLoggedIn(self):
                     return True
         return False
 
+    def __updateGpuAndLlu(self, frame):
+        if 'GPU_LIST' in frame.runFrame.attributes:
+            usedGpuMemory = 0
+            for unitId in frame.runFrame.attributes.get('GPU_LIST').split(','):
+                usedGpuMemory += self.getGpuMemoryUsed(unitId)
+
+            frame.usedGpuMemory = usedGpuMemory
+            frame.maxUsedGpuMemory = max(usedGpuMemory, frame.maxUsedGpuMemory)
+
+        if os.path.exists(frame.runFrame.log_dir_file):
+            stat = os.stat(frame.runFrame.log_dir_file).st_mtime
+            frame.lluTime = int(stat)
+
     def rssUpdate(self, frames):
         """Updates the rss and maxrss for all running frames"""
+        if platform.system() == 'Windows' and winpsIsAvailable:
+            values = list(frames.values())
+            pids = [frame.pid for frame in list(
+                filter(lambda frame: frame.pid > 0, values)
+            )]
+            # pylint: disable=E1101
```

Comment:
Let's use the name of the rule being disabled instead of the ID -- IMO it makes it more easily understood and is consistent with the rest of the code. I think this one is `no-member`, right?